### DISCUSSION TITLE
Reset fan curve?

### DISCUSSION THREAD

**Discussion by boutzamat** [Rem0o/FanControl.Releases#2010] [Q&A]
I installed this software but it seems a bit too advanced for my usecase. I messed around with the GPU fans. Since there is no "uninstall" option, how do i reset the fans back to standard?
I have a founders edition card, so i can't use Nvidias own software to set its fan curve to "auto" as it is from the factory.

**Comment by Rem0o** [comment]
Simply exiting the program will reset it back.
